api/models/Membership.js

```diff
@@ -91,7 +91,10 @@ module.exports = bookshelf.Model.extend(merge({
     return bookshelf.knex('communities_users').where({
       user_id: user_id,
       community_id: community_id
-    }).update({last_viewed_at: new Date()})
+    }).update({
+      last_viewed_at: new Date(),
+      new_post_count: 0
```

Comment:
This is happening in a query, not a mutation, but I think the same argument for updating last_viewed_at applies here.